svn/common.py

```diff
@@ -49,10 +49,13 @@ def run_command(self, subcommand, args, success_code=0,
 
         _logger.debug("RUN: %s" % (cmd,))
 
+        environment_variables = os.environ
```

Comment:
I'm sorry for commenting not my pull request sent not to me, but shouldn't you copy environment to prevent its mutation?